core/src/main/thrift/tabletserver.thrift

```diff
@@ -164,8 +174,9 @@ service TabletClientService extends client.ClientService {
                                   5:map<string, map<string, string>> ssio,
                                   6:list<binary> authorizations,
                                   7:bool waitForWrites,
-                                  9:i64 batchTimeOut)  throws (1:client.ThriftSecurityException sec),
-  data.MultiScanResult continueMultiScan(2:trace.TInfo tinfo, 1:data.ScanID scanID) throws (1:NoSuchScanIDException nssi),
+                                  9:TSamplerConfiguration samplerConfig,
```

Comment:
I think thats ok.  batchTimeOut is new in 1.8.0-SNAPSHOT.  At some point I merged it in and I had a conflict and thats how I resolved it.